### DISCUSSION TITLE
CRF in cppflow

### DISCUSSION THREAD

**Discussion by MaxMalmer** [serizba/cppflow#217] [Q&A]
Hey everyone, hope your having an awesome day!

I was wondering if anyone has implemented a CRF wrapper implementation in cppflow? I was thinking of doing it myself as I can't seem to find an open source solution.

**Comment by serizba** [comment]
Hi @MaxMalmer 

What do you want exactly? Couldn't a model with a CRF created in python be opened in cppflow?

**Reply by MaxMalmer** [reply]
That's true! I am using it in a way where I set dimensions and only load the weights, so that's why I need a wrapper where I only load the weights and set the feature dimensions and a layer of neurons.

**Comment by serizba** [comment]
I think that still, the easiest way to go is to load a model created with python. If you have the weights that means that you already trained the model, right? Thus you could save it. The cppflow wrapper is quite lightweight and thus only let you do basic tensor operations and load models.

**Reply by MaxMalmer** [reply]
Sounds like a plan, I think I've got a way to solve that in program as well such that I do many calls to predict from the model rather than defining it in different ways. 

Thanks for the help!
